server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -551,6 +613,7 @@ export class DeliLambda extends EventEmitter implements IPartitionLambda {
                     });
 
                     const controlContents = controlMessage.contents as IUpdateDSNControlMessageContents;
+                    this.serviceSummaryGenerated = !controlContents.isClientSummary;
```

Comment:
Can you use `this.durableSequenceNumber > 0` and `this.canClose` instead of needing a new variable?

Reply:
please correct me if I wrong but this.dsn would be > 0 as long as there was a successful summary (client or service) right? canClose seems to be making sure that there are no active clients but the two conditions together do not necessarily verify that a "service summary" was created successfully. They only seem to verify that there was at least one successful summary and the quorum dropped to 0.

Reply:
That's right. This is good than.